### PR TITLE
bug: Stop splitting version info when logging for websocket

### PR DESCRIPTION
**Pull request by jrconlin**
fixes: #693

### CODE REVIEW

**Comment by jrconlin** [comment]
Actual change is a one-liner at https://github.com/mozilla-services/autopush/pull/694/files#diff-eff17175be11d1f3b2faddd0c7c93cd9L1246, the other changes sure up unit testing by using "real" values. This should prevent similar issues in the future.

File: autopush/tests/test_websocket.py [hunk 1]
```diff
@@ -31,6 +31,28 @@
 from autopush.utils import base64url_encode, ms_time
 
 
+dummy_version = u'gAAAAABX_pXhN22H-hvscOHsMulKvtC0hKJimrZivbgQPFB3sQAtOPmb' \
```
Comment (pjenvey) [inline autopush/tests/test_websocket.py]: fyi (don't care about changing it) you can surround these in parens to get the auto concatenate w/out end of the line \s.

or a = u"""\<newline>gAA<...>OX8=""".replace("\n", "") works pretty nice for blocks like these

Comment (jrconlin) [inline autopush/tests/test_websocket.py]: I'll admit that I cheated and let pycharm do the line breaks.

Comment (jrconlin) [inline autopush/tests/test_websocket.py]: ✅

**Pull request merged**